### ISSUE TITLE
git-sdk-installer-1.0.8-64.7z.exe does not install bin/bash.exe any more

### ISSUE THREAD

**Issue by Dirk1966** [git-for-windows/git#3594]
- [x] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Start 64-bit git-sdk-installer-1.0.8-64.7z.exe

```
 - Windows 10-64?

```
$ cmd.exe /c ver

** Microsoft Windows [Version 10.0.19044.1387] **
```

 - What options did you set as part of the installation? Or did you choose the
   defaults? No I choosed different paths on a secondary hard disk P:\git-sdk-64, P:\Prg\t1\git-sdk-64

```

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

  Searching bash.exe should show three files, the git-sdk-64/bin/bash.exe is missing:

  Previous installations:

** find /p/Prg/git-sdk-64 -type f -name 'bash.exe' -exec ls -l '{}' \;
** -rwxr-xr-x 1 user 197121 46096 Nov 26 21:37 /p/Prg/git-sdk-64/bin/bash.exe  # This file is missing since at least 2021-12-13
** -rwxr-xr-x 1 user 197121 1089162 Aug 26 18:26 /p/Prg/git-sdk-64/mingw64/libexec/busybox/bash.exe
** -rwxr-xr-x 1 user 197121 1963639 Aug 26 18:27 /p/Prg/git-sdk-64/usr/bin/bash.exe

### Details

 - What commands did you run to trigger this issue?

** .\git-sdk-installer-1.0.8-64.7z.exe

 - What did you expect to occur after running these commands?
 
** A file .../git-sdk-64/bin/bash.exe exists **

 - What actually happened instead?

** File .../git-sdk-64/bin/bash.exe missing in installation **

**Comment by dscho** [comment]
I don't think we ever installed [the Git wrapper](https://github.com/git-for-windows/git/wiki/Git-wrapper) into `<sdk>\bin`. It _is_ installed as part of the installer, though, and even the portable Git has it there, but not the SDK.

**Comment by Dirk1966** [comment]
It was installed always for at least two years up until the beginning of December 2021.
Is there a possibility to install the Git wrapper afterwards?
I prefer the git for Windows SDK because then I do not need git for Windows itself.

**Comment by dscho** [comment]
> I prefer the git for Windows SDK because then I do not need git for Windows itself.

Right. I do have enough on my hands supporting Git for Windows, though ;-)

**Comment by dscho** [comment]
> It was installed always for at least two years up until the beginning of December 2021.

@Dirk1966 If you have time to investigate a bit more, please do so.

**Comment by Dirk1966** [comment]
Hi,
I use a workaround to get this installed again:
`pacman -S mingw-w64-x86_64-git-doc-html`
Best regards,
Dirk (Dirk1966)

**Comment by dscho** [comment]
@Dirk1966 But how come that this is not included in the installer anymore? Do we need to run something like

```
git -c alias.x="!. /var/lib/pacman/local/mingw-w64-x86_64-git-[0-9]*/install && post_install" x
```

after [this line](https://github.com/git-for-windows/build-extra/blob/e7c8277a3b03ea987593f1539c324c4c14867cc3/sdk-installer/setup-git-sdk.bat#L26)?

**Comment by Dirk1966** [comment]
Sorry, never built the SDK myself, do not know where to look for.

**Comment by PhilipOakley** [comment]
The SDK installer is in the build-extra repo [here](https://github.com/git-for-windows/build-extra/tree/main/sdk-installer). 

It (when installing) does a bit of bootstrapping that clones, depth=1, the appropriate SDK setup e.g. [git-sdk-64](https://github.com/git-for-windows/git-sdk-64). 

Those SDKs are 'pacman ready' in that they are kept up to date using/matching pacman, so local updates are via pacman, rather than more fetches.

What is released [here ](https://github.com/git-for-windows/build-extra/releases/tag/git-sdk-1.0.8) is actually a 7-zip exe that does the bootstrapping. 

Hopefully that's will get you into the ball park to start looking at the right bits of code.
